src/EnergyPlus/Coils/CoilCoolingDX.cc

```diff
@@ -416,7 +458,7 @@ void CoilCoolingDX::size() {
     this->performance.size();
 }
 
-void CoilCoolingDX::simulate(bool useAlternateMode, Real64 PLR, int speedNum, Real64 speedRatio, int fanOpMode)
+void CoilCoolingDX::simulate(int useAlternateMode, Real64 PLR, int speedNum, Real64 speedRatio, int fanOpMode, Real64 LoadSHR)
```

Comment:
The bool to int change is expected.  The addition of LoadSHR is not necessarily expected, but I did figure _something_ would need to be passed in.